components/ImageR/imageR.html

```diff
@@ -0,0 +1,228 @@
+          </div>
+
+          <div class="divider-space"></div>
+
+          <div class="content-header">
+           Round Images
+          </div>
+
+          <div class="content-subheading">
+            Use class <strong>.round-image</strong> for round images.
+          </div>
+
+          <div class="example-content ">
+            <img src="/images/harry-image.PNG "
+            alt="image" class="round-image" />
+          </div>
+
+          <iframe
+            src="https://carbon.now.sh/embed/6o7KYmefdHgUzhd2jWJK"
+            style="
+              width: 50rem;
+              height: 15rem;
+              margin-top: 2rem;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+
+          <div class="divider-space"></div>
+
+          <div class="content-header">
+           Square Images
+          </div>
+
+          <div class="content-subheading">
+            Use class <strong>.square-image</strong> for square images.
+          </div>
+
+          <div class="example-content ">
+            <img src="/images/harry-image.PNG  "
+            alt="image" class="square-image" />
+          </div>
+
+          <iframe
+            src="https://carbon.now.sh/embed/uxUUtzNYRMRNvEbQyoo3"
+            style="
+              width: 50rem;
+              height: 15rem;
+              margin-top: 2rem;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+
+          <div class="divider-space"></div>
+
+          <div class="content-header">
+           Responsive Images
+          </div>
+
+          <div class="content-subheading">
+            Use class <strong>.responsive-image</strong> for responsive images.
+          </div>
+
+          <div class="example-content ">
+            <img src="/images/harry-image.PNG "
+            alt="image" class="responsive-image" />
+          </div>
+
+          <iframe
+            src="https://carbon.now.sh/embed/vXUXgacUwDA7CWgwFXKX"
+            style="
+              width: 50rem;
+              height: 15rem;
+              margin-top: 2rem;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+
+      </div>
+    </div>
+    <footer class="footer">
+      <div class="footer-header">Made by Shivangi Sharma</div>
+      <ul class="footer-list">
+        <li class="list-item-inline">
+          <a
+            class="links"
+            href="https://github.com/Shivangisharma01"
+            target="blank"
```

Comment:
are you planning to open in the same window since you are using just blank, it will also work

Reply:
The target blank will open this link in new tab, that's why I've used it.

Reply:
target:_blank opens up in new window and blank opens up in the same window,do check it once